ecc-functions/entities-properties/__init__.py

```
@@ -0,0 +1,35 @@
+import os, json, urllib.request, urllib.parse, azure.functions as func, os as _os
+
+SUPABASE_URL = os.environ.get("SUPABASE_URL")
+SUPABASE_KEY = os.environ.get("SUPABASE_SERVICE_ROLE_KEY")
+TABLE = _os.path.basename(_os.path.dirname(__file__)).split("-")[-1]
+
+def _sb(path, params):
+    if not SUPABASE_URL or not SUPABASE_KEY:
+        raise RuntimeError("Missing SUPABASE_URL or SUPABASE_SERVICE_ROLE_KEY")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)